src/lplib/cmap.ml

```diff
@@ -0,0 +1,174 @@
+(*Type of points and intervals. Used to determine if a cursor is in the range
```

Comment:
Use ocamldoc features for documentation like `(**`, etc. Please check ocamldoc documentation and check the result by doing `make doc` (I believe). If you have questions on ocamldoc, please ask @gabrielhdt

Reply:
Mostly solved.